addons/medical_gui/functions/fnc_onKeyDown.sqf

```diff
@@ -0,0 +1,84 @@
+#include "..\script_component.hpp"
+#include "\a3\ui_f\hpp\defineDIKCodes.inc"
+/*
+ * Author: AmsteadRayle
+ * Handles keyboard inputs in medical menu.
+ *
+ * Arguments:
+ * 0: Menu display <DISPLAY>
+ * 1: Key being pressed <NUMBER>
+ * 2: Shift state <BOOL>
+ * 3: Ctrl state <BOOL>
+ * 4: Alt state <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+*/
+// TODO: Is the airway category ever visible? Can the dynamic category stuff be removed?
```

Comment:
It will be eventually, plus this maintains support for KAM. Keep it.

Reply:
What is KAM?

Reply:
3rd party medical addon